lib/http.js

```diff
@@ -47,7 +50,9 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
             actor,
             account: {
               id: `urn:uuid:${uuid()}`,
-              controller
+              controller,
+              // add seed to account so a controller key can be used
+              controllerKeySeed: base64url.encode(crypto.randomBytes(32)),
```

Comment:
Just to make sure, this has previously been done in an `account.postInsert` method. I'm assuming this is fine, but just wondering.